scrunch/tests/test_datasets.py

```diff
@@ -1795,7 +1846,8 @@ def test_fork_preserve_owner(self):
             'body': {
                 'name': 'FORK #1 of ds name',
                 'description': 'ds description',
-                'owner': user_id,  # Owner preserved
+                # 'owner': user_id,  # Owner preserved
+                'owner': project_id,
```

Comment:
I think it's best to start updating the tests to use `project` to make sure anybody else following the implementation starts learning this is the preferred way

Reply:
Updated

Reply:
Hi @jjdelc ,

I've updated the `try-catch` block to set timeout=None to avoid getting timed out again & have also added a comment stating the purpose.

Reply:
I continue to worry that this re-attempt strategy is a bad approach. What this is doing is capturing the 1st timeout and then trying again. 

Recall that this is a client-side timeout, say 60s so in case the opreation in the backend ends after 61s, the client will time out, and raise an error, then the client will try again a 2nd time and proceed for a next creation.

I think that if the issue is to reduce timeout changes, this should simply set a long timeout on the very fist request. Instead of making 2 independent creation requests to the server.

Reply:
Hi @jjdelc,

As we've already completed the main goal of this PR, which involves allowing Project paths & URL, 
I think it would be a good idea to explore timeout handling separately. 

Therefore, I've removed the timeout handling part from the `fork()` method as this step is just an extra precaution to handle timeouts, but it's not the primary focus of this PR.

Reply:
Very good call @shaikh-ma , I think this is best, if the fork starts to timeout, we can address that later.